src/Setup.Page.al

```diff
@@ -22,7 +22,6 @@ page 50102 "ENVHUB Setup"
                 field("Client ID"; ClientID)
                 {
                     ApplicationArea = All;
```

Comment:
Please don't remove the ExtendedDatatype = Masked;
For security reasons we have descided to leave this one masked.

Reply:
Masking for ClientID has been restored.